package.json

```diff
@@ -31,7 +31,6 @@
     "build:client:live": "webpack ./client-src/live/index.js -o client/live.bundle.js --color --config client-src/live/webpack.config.js",
     "build:client:sockjs": "webpack ./client-src/sockjs/index.js -o client/sockjs.bundle.js --color --config client-src/sockjs/webpack.config.js",
     "build:client": "rimraf ./client/* && npm-run-all -s -l -p \"build:client:**\"",
-    "webpack-dev-server": "cd $INIT_CWD && node ../../../bin/webpack-dev-server.js",
```

Comment:
Don't delete this. This is for samples.

Reply:
@hiroppy it doesn't worj on windows, also we need this line when we don't have tests (for manual testing), but right now we have many tests and good helpers, so i think it is legacy trash and we can remove this safely

Reply:
> node ../../../bin/webpack-dev-server.js",

also is this path working? 
if we are keeping the script then I guess we need to consider `cross-env`

Thoughts ?

Reply:
cc @evilebottnawi @hiroppy

Reply:
> it doesn't work on windows, also we need this line when we don't have tests (for manual testing), but right now we have many tests and good helpers, so i think it is legacy trash and we can remove this safely

How to build the examples? We need this line.

Reply:
@hiroppy hm, do you use this command? I just use `node bin/webpack-dev-server` in command line

Reply:
@evilebottnawi Yes, because README says to use it.😅 e.g. https://github.com/webpack/webpack-dev-server/tree/master/examples/cli/bonjour

Reply:
>  We need this line. If this line doesn't work on Windows, we should change this line.

the `cd $INIT_CWD` should be changed with `cd %INIT_CWD%` if it's in windows! but then it won't work in other OS 😬 . 
what can be done? `cross-env` ? but I don't think its worth using and increasing dependency just for this !

Reply:
> but I don't think its worth using and increasing dependency just for this !

I don't think so, because we have already had many examples. If you delete this line, we should change all examples build style. I recommend adding `cross-env` as devDependencies if we want to run on Windows.

Reply:
okay ! it seems we can add it then !

Reply:
Will add a seperate PR for that, we can discuss clearly there I guess

Reply:
@hiroppy done ! 👍

Reply:
@anikethsaha I'm not sure if you were working on this but I couldn't get `cross-env` to do this successfully on Windows so I wrote my own solution here: https://github.com/webpack/webpack-dev-server/pull/2112

Reply:
Yeah cross env was not the working for me too.